src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/ManageIncentiveReviewsResource.kt

```diff
@@ -122,51 +119,6 @@ class ManageIncentiveReviewsResource(
     id: Long,
   ): IncentiveReviewDetail = prisonerIncentiveReviewService.getReviewById(id)
 
-  @PostMapping("/bookings")
-  @Operation(
```

Comment:
This has not been used for over 2 years

Reply:
was it used for reconciliation or something? 🤔